js/scripts.js

```diff
@@ -1,3 +1,9 @@
-var name = prompt('Enter your name');
-alert('Hello, ' + name);
-console.log('Hello, ' + name);
+// var name = prompt('Enter your name');
+// alert('Hello, ' + name);
+// console.log('Hello, ' + name);
+
+var text = ('Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.');
+var dinosaur = 'triceratops';
+var nameUpperCased = dinosaur.toUpperCase();
+var changeWord = text.replace('Velociraptor', nameUpperCased);
```

Comment:
Twój program zadziała tylko dla tego konkretnego zdania. Jeśli `text` ulegnie zmienie, to już nie wyświetlisz idealnie połowy tekstu. Spróbuj zrobić to tak żeby tekst dzielić na pół bez względu na to jak długi on będzie.
Zrób tak żeby zamiast Velociraptor , zamieniany był zawsze pierwszy wyraz zmiennej `text` - bez względu na to co tam będzie. Możesz użyć do tego metody `split` (google -> "mdn split")